tools/experiments/experiments.js

```diff
@@ -235,12 +235,6 @@ const EXPERIMENTS = [
     name: 'Use slot width/height attribute for AdSense size format',
   },
   {
-    id: 'amp-form-verifiers',
```

Comment:
changes to `prod-config.js` and `canary-config.js` must follow code changes after 1 release cycle. (we have presubmit checks for that). this one is fine .